src/main/java/uk/gov/hmcts/reform/divorce/orchestration/workflows/NotifyRespondentOfDARequestedWorkflow.java

```diff
@@ -10,25 +10,36 @@
 import uk.gov.hmcts.reform.divorce.orchestration.framework.workflow.DefaultWorkflow;
 import uk.gov.hmcts.reform.divorce.orchestration.framework.workflow.WorkflowException;
 import uk.gov.hmcts.reform.divorce.orchestration.framework.workflow.task.Task;
-import uk.gov.hmcts.reform.divorce.orchestration.tasks.SendDaRequestedNotifyRespondentEmail;
+import uk.gov.hmcts.reform.divorce.orchestration.tasks.SendDaRequestedNotifyRespondentEmailTask;
 
 import java.util.Map;
 
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.APPLY_FOR_DA;
 import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.CASE_DETAILS_JSON_KEY;
 
 @Component
 public class NotifyRespondentOfDARequestedWorkflow extends DefaultWorkflow<Map<String, Object>> {
 
     @Autowired
-    private SendDaRequestedNotifyRespondentEmail sendDaRequestedNotifyRespondentEmail;
+    private SendDaRequestedNotifyRespondentEmailTask sendDaRequestedNotifyRespondentEmailTask;
 
     public Map<String, Object> run(final CcdCallbackRequest ccdCallbackRequest) throws WorkflowException {
         CaseDetails caseDetails = ccdCallbackRequest.getCaseDetails();
 
+        validateDaIsRequested(caseDetails);
+
         return this.execute(
-            new Task[] {sendDaRequestedNotifyRespondentEmail},
+            new Task[] {sendDaRequestedNotifyRespondentEmailTask},
             ccdCallbackRequest.getCaseDetails().getCaseData(),
             ImmutablePair.of(CASE_DETAILS_JSON_KEY, caseDetails)
         );
     }
+
+    private void validateDaIsRequested(CaseDetails caseDetails) throws WorkflowException {
+        String applyForDecreeAbsolute = (String) caseDetails.getCaseData().get(APPLY_FOR_DA);
+
+        if (applyForDecreeAbsolute == null || !applyForDecreeAbsolute.equalsIgnoreCase("yes")) {
```

Comment:
```suggestion
        if (YES_VALUE.equalsIgnoreCase(applyForDecreeAbsolute)) {
```
Note YES_VALUE is "Yes"

Reply:
cheers